File: Ej 32 (Bisiestos)/Java/Bisiestos.java
/**
 * @author DeathbatO
 */
import java.util.Scanner;
public class Bisiestos{
    public static void main(String[] args) {
        var sc = new Scanner(System.in);
        int year, count = 0;
        System.out.print("Introduce un año: ");  year = sc.nextInt();
        sc.close(); System.out.println("Los proximos 30 años bisiestos son: ");
        while(count < 30){
            year++;
            if((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)){
                System.out.println(year);
            }
        }
    }
}

File: Ej 32 (Bisiestos)/Python/Bisiestos.py
## DeathbatO
count = 0
year = int(input("Introduce un año: "))
print("Los proximos 30 años bisiestos son: ")
while count < 30:
    year += 1
    if (year % 4 == 0 and year % 100 != 0) or (year % 400 == 0):
        print(year)
        count += 1